### ISSUE TITLE
Incorrect possessive case for usernames ending in "s"

### ISSUE THREAD

**Issue by mgartner** [robinjoseph08/pokedextracker.com#133]
<img width="373" alt="screenshot 2016-05-06 15 56 34" src="https://cloud.githubusercontent.com/assets/1128750/15088008/5508eba4-13a3-11e6-8635-a5e3de41e580.png">

http://www.grammar.cl/rules/genitive-case.htm

**Comment by ami** [comment]
@mgartner according to that article,

#### 4. Singular noun ending in –s:
It depends…

a. Most names: add  's (apostrophe S)
> They had a really good time at James’s barbecue last Friday.
> We spent the day admiring Frances’s new car.

b. Classical or religious names: add ' (only the apostrophe)
> Jesus’ disciples carried out the teachings of Jesus.
> Sophocles’ plays are still performed today.

are you a classical or religious figure? iiiiiidon'tthinksooooo

**Comment by robinjoseph08** [comment]
TL;DR are you Jesus, @mgartner?

**Comment by mgartner** [comment]
I guess not. But this guy is a classic figure: https://en.wikipedia.org/wiki/Marcus_Aurelius

How is he supposed to track his Pokémon?
